### DISCUSSION TITLE
diskFillFraction does not work

### DISCUSSION THREAD

**Discussion by JimStorage** [minio/minio#14474] [Q&A] [answered]
in file: cmd/global.go  , "diskFillFraction = 0.99"  is upper limit of the fraction of a disk allow to be filled
but In my real environment, disk space is used up.
Who could told the reason, I will be appreciated.

**Comment by klauspost** [comment, marked as answer]
@JimStorage This is only enforced in erasure coded mode. Without more info that is my guess.

**Reply by JimStorage** [reply]
I think it's running in erasure coded mode.

//each storagenode is a physical server node, each storagedisk is a physical HDD
startup parameters:  minio server http://storagenode{1...4}/data/storagedisk{1...12}    
MINIO_STORAGE_CLASS_STANDARD="EC:2"
OS version: Ubuntu 20.04.3 LTS (GNU/Linux 5.4.0-96-generic x86_64)

linux cmd "df":
/dev/sda1        7811938304   7811938068  236  100%  /data/storagedisk1                  //236 is Available capacity

**Reply by klauspost** [reply]
@JimStorage The fill-disk fraction is not a sleeping pillow. You should monitor your disk use.

MinIO uses storage for transient data, so you can easily get beyond 99%.
